improver_tests/utilities/ancillary_creation/test_SaturatedVapourPressureTable.py

```diff
@@ -53,6 +53,20 @@ def test_basic(self):
         self.assertEqual(result, msg)
 
 
+class Test_saturation_vapour_pressure_goff_gratch(IrisTest):
+
+    """Test calculations of the saturated vapour pressure using the Goff-Gratch
+    method."""
+
+    def test_basic(self):
+        """Basic calculation of some saturated vapour pressures."""
+        data = np.array([[260., 270., 280.]], dtype=np.float32)
+        plugin = SaturatedVapourPressureTable()
+        result = plugin.saturation_vapour_pressure_goff_gratch(data)
+        expected = 0.01*np.array([[195.6419, 469.67078, 990.9421]])
+        self.assertArrayAlmostEqual(result, expected)
```

Comment:
Moved from `improver_tests.psychrometric_calculations.test_Utilities` with no modification to input values.